how-to/about-how-to.md

```diff
@@ -1,62 +1,57 @@
-# The how-to article
+# The How To article
 
-## When do I need a how-to article?
+How To articles take the reader through a series of steps required to solve a specific problem.
+They explain in detail how to do a specific task.
+Don't use How To articles to teach concepts, but to prescribe a set of sequential steps readers must do to be able to use a feature.
 
-How-to articles take the reader through a series of steps required to solve a specific, real-world problem.
-Think of them as recipes for getting stuff done!
+How To articles are often confused with tutorials. How To articles are problem-oriented, while tutorials are learning-oriented.
 
-Sometimes you need to explain in detail how to do a specific task to a user.
-You don't need to teach them concepts, but you do need to prescribe a set of sequential steps they must do in that order otherwise they won't be able to use a feature properly.
+Readers that have experience with the product and need a specific question answered use How To articles.
+New users who want to solve a problem instantly can also benefit, provided they are well-written and state any prerequisite knowledge required to complete the task.
 
-How-to articles are often confused with tutorials, which is not the case as they are _problem_-oriented not _learning_-oriented.
+## Content of your How To article
 
-How-to articles are used by those who have experience with the product and need a specific question answered.
-New users who want to solve a problem instantly can also benefit from how-to articles provided they are well-written and clearly state any prerequisite knowledge required to complete the how-to.
+### The Overview section
```

Comment:
I wonder if we should adopt the pattern

`### The "Overview" section` 

To make the heading in this "about" document clear that it refers to the heading with casing preserved.

Reply:
A lot of the titles used quotation marks around template titles, and I took them out. GDSG doesn't give specific guidance on this, but I tend to err on the side of less punctuation in titles generally.